styles/style.css

```
@@ -0,0 +1,168 @@
+header #navigation {
+    position: relative;
+    width: 100%;
+    height: 70px;
+    line-height: 80px;
+    border-bottom: 1px solid rgb(255 255 255 /30%);
```

I don't think the `%30` here is valid CSS.

```suggestion
    border-bottom: 1px solid rgb(255, 255, 255);
```